### PR TITLE
use relative link paths in propose_a_new_market

### PR DESCRIPTION
**Pull request by 2dpetkov**
Using relative link paths in `propose_a_new_market` will help other repository forks to exist independently without having fix conflicts when merging.

This approach has been used previously, for example in `required_node_configs`, `voting`, `submitting_a_proposal`.

### CODE REVIEW

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 1]
```diff
@@ -8,10 +8,9 @@ dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipien
 
 Example:
 ```bash
-NATIVE_TOKEN_DENOM=adv4tnt
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:11]: Changes in this file should be resolved now by [this PR](https://github.com/dydxprotocol/v4-documentation/pull/114). Could you revert these changes?

I agree that we can make the changes in pages/governance/proposing_a_new_market.md

**Review by ttl33 — APPROVED** [review]

File: pages/deposits_and_withdrawals/cli_commands.md (hunk 1 above)
Comment (2dpetkov) [reply pages/deposits_and_withdrawals/cli_commands.md:11]: hey, @ttl33 
PR #114 still uses full path to docs.dydx.exchange - is this on purpose?

Comment (2dpetkov) [reply pages/deposits_and_withdrawals/cli_commands.md]: Now resolved, only the changes in `propose_a_new_market` should be reviewed
